frontend/src/metabase/components/CollectionLanding.jsx

```diff
@@ -17,6 +17,8 @@ import Icon from "metabase/components/Icon";
 import Link from "metabase/components/Link";
 import CollectionEmptyState from "metabase/components/CollectionEmptyState";
 import EntityMenu from "metabase/components/EntityMenu";
+import Subhead from "metabase/components/Subhead";
+import Ellipsified from "metabase/components/Ellipsified";
```

Comment:
Out of curiosity, have you looked at how Ellipsified and Truncate differ in behavior and/or implementation?